src/Contract.tsx

```diff
@@ -156,7 +156,7 @@ export function ContractManager(props: ModalProps) {
                                     </Row>
                                     <Row>
                                         <Col sm={2}>Balance:</Col>
-                                        <Col sm={10}>{currentContract.amount.microGtuAmount.toString()} μCCD</Col>
+                                        <Col sm={10}>{currentContract.amount.toString()} μCCD</Col>
```

Comment:
```suggestion
                                        <Col sm={10}>{currentContract.amount.toJSON()} μCCD</Col>
```
The toString() just returns the good old [object Object]
(Or do `currentContract.amount.microCcdAmount.toString()` )

Reply:
Dang I was certain that this had been implemented as `toString`. Nice catch, will revert.